### DISCUSSION TITLE
[Extended Sync] How does fetching data from the provider using EDR work?

### DISCUSSION THREAD

**Discussion by anschl** [eclipse-tractusx/tractusx-edc#1144] [Q&A]
As described in the [tutorial](https://github.com/eclipse-tractusx/tutorial-resources/blob/main/mxd/docs/EDR%20Transfer%20Tutorial.md#fetching-data), there are two options for fetching data requested via EDR.

One of them is using `http://localhost/alice/api/public`, called "Get Data with provider dataplane" in the [Postman collection](https://github.com/eclipse-tractusx/tutorial-resources/blob/main/mxd/postman/mxd-management-apis.json). The request returns data but I don't understand how it works. As there are no GET parameters and there's also no request body, how do I define what I want to receive? I also can't get it to respond with anything else than this:
```
{
  "userId": 1,
  "id": 1,
  "title": "delectus aut autem",
  "completed": false
}
```

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
